File: src/numerical/RamanujanNumbers/test_ramanujan_numbers.py
from ramanujan_numbers import get_ramanujan_numbers
from time import time

thresholds = [50, 100, 150]
output = ''
for threshold in thresholds:
    start = time()
    numbers = get_ramanujan_numbers(threshold)
    end = time()
    exec_time = int((end - start) * 1000)
    output += f'Для чисел меньших {threshold} найдено '\
        f'{len(numbers)} чисел Рамануджана\n'
    output += f'Время выполнения - {exec_time}ms\n\n'
print(output)

File: src/numerical/RamanujanNumbers/TestRamanujanNumbers.java
package numerical.RamanujanNumbers;

import static numerical.RamanujanNumbers.RamanujanNumbers.getRamanujanNumbers2;

import java.io.FileWriter;
import java.io.IOException;
import java.nio.charset.StandardCharsets;
import java.util.List;

public class TestRamanujanNumbers {
    public static void main(String[] args) throws IOException {
        int threshold = 1000;
        StringBuilder builder = new StringBuilder();

        long start = System.currentTimeMillis();
        List<List<Long>> sums = getRamanujanNumbers2(threshold);
        long end = System.currentTimeMillis();
        builder.append(String.format(
                "Для чисел меньших %d найдено %d чисел Рамануджана%n",
                threshold, sums.size()));
        builder.append(String.format(
                "Время выполнения - %dms%n%n",
                end - start));

        FileWriter writer = new FileWriter(
                "./src/numerical/RamanujanNumbers/java_test.txt",
                StandardCharsets.UTF_8);
        writer.write(builder.toString());
        for (List<Long> list : sums) {
            writer.write(list.toString() + System.lineSeparator());
        }
        writer.close();
    }
}